### ISSUE TITLE
Add support for aria-label/labelled-by

### ISSUE THREAD

**Issue by williamernest** [tajo/react-range#147]
New a11y guidelines indicate that all elements with role="slider" should support a title, label or labelledby attribute as an author provided name is now required. 

https://accessibilityinsights.io/info-examples/web/aria-input-field-name/

More details on the newest changes here
https://www.w3.org/TR/wai-aria-1.2/#namecalculation

**Comment by BogdanCerovac** [comment]
Interactive elements need to have accessibility name for multiple assistive technologies like screen readers.

I would like to point out that this issue should be considered as critical and not enhancement - as I went quickly through the code and I don't actually see an existing naming prop, this means that all the sliders are failing Web Content Accessibility Guidelines (specifically  WCAG 4.1.2 Name Role Value (Level A) - understanding docs: https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

Simplest possible fix is, as mentioned by @williamernest, adding the `aria-label="Loan or something"` to the slider (on same level as we now have the `role="slider"`. That would help a lot.

If we have a variant where label is visible (cloning `<label for="slider">Loan</slider><input type="range" id="slider"...>`), then we can use `aria-labelledby="${idOfVisibleLabel}"` on the slider and a unique id on the visible label, like for example here:

```
<p id="1243fg34q545">Loan</p><!-- important that ID value is unique per element -->
<div role="slider" aria-labelledby="1243fg34q545" ...></div>
```

More info: https://www.w3.org/WAI/ARIA/apg/patterns/slider/examples/slider-color-viewer/

**Comment by tajo** [comment]
@BogdanCerovac do you want to contribute?

**Comment by BogdanCerovac** [comment]
@tajo - I had some extra time on my hands and made a quick fix just to fix this critical issue.

Please check PR: https://github.com/tajo/react-range/pull/199

When I have more time I am planning to test more and perhaps add additional changes, but this fixes the critical issue of not being to able the element.
